lib/driver.rb

```diff
@@ -43,5 +43,34 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def calculate_driver_revenue
+      fee = 1.65
+      driver_takehome = 0.8
+
+      subtotal = 0
+      @trips.each do |trip|
+        subtotal += trip.cost - fee
+      end
+      total = subtotal * driver_takehome
+      return total
+    end
+
+    def calculate_driver_average_revenue
+      return calculate_driver_revenue / @trips.length
+    end
```

Comment:
This method is supposed to be the driver's average _hourly_ revenue, that is, the total revenue divided by the number of hours driven, not by the number of trips.